### ISSUE TITLE
No response from developer

### ISSUE THREAD

**Issue by Anutrix** [wummel/patool#58]
I think someone should fork this and maintain in a new repository separately since the developer seems to have abandoned this project. No response since a year.
If the developer is still working on this I am sorry. Please respond to issues.

**Comment by yarikoptic** [comment]
yeah, seems to be that @wummel is moved on... but what I forgot is that in 2013 he kindly added me to Contributors for the patool, so I have some God's powers for this repository, which I could exercise.  But I do not see pull requests piling up - forking on its own wouldn't fix issues somehow automagically.  If you are interested to contribute @Anutrix , I will try to be responsive to review suggested changes etc, but I will need to pay extra caution since I am not too familiar with the codebase. (may be @kyleam would be interested to give a hand as well)
